data/explorations/audio_test/audio_test.yaml

```diff
@@ -17,11 +17,11 @@ states:
       audio_translations:
         en:
           filename: 'test_audio_2_en.mp3'
-          file_size_bytes: 2
+          file_size_bytes: 100
```

Comment:
Consider using the actual values for the files (59 in this case; ditto below). (I noticed this when trying to do a sanity check of the 1.13 MB.)